src/oauth2-client/index.ts

```diff
@@ -72,59 +72,6 @@ abstract class OAuth2Client {
     return url.toString();
   }
 
-  /**
-   * DEPRECATED
-   * Calls the authorize URL with an iframe. If successful,
-   * it returns the callback URL with authentication code,
-   * optionally using PKCE.
-   */
-  public static async getAuthorizeUrl(options: GetAuthorizationUrlOptions): Promise<string> {
```

Comment:
This function needs to be preserved, but we need to rename it. The problem with this name was that it was misleading. The calling of this method does not get the authorize URL itself and return it. The calling of this method returns the result of the request to `/authorize`. So, we could name it something like `authorizeByIframe`, and I think that would clear this confusion up. Thoughts?

Reply:
OK Now it makes more sense. I was going with the doc here:
https://docs.google.com/spreadsheets/d/1_TW6F55PAp52eOqUoZIWYyR7MSDDyEg3fs-v8oWuMEc/edit#gid=0
That said remove, so went and nuked it from orbit.

"authorizeByIframe" or "authorizeWithIframe", "authorizeForLegacy"

All sound good to me. Will revisit and reinstate that. Please let me know what is your preference for name

Reply:
Sorry, that doc should have said "rename". That's my bad.

As for names, either of the first two you mentioned works for me. There's also `getAuthCodeByIframe` or `getAuthCodeLegacy` that could work. I do like how `getAuthCode` is much more specific in its purpose. I'll leave it to you to name it :)